tools/system_calls.py

```diff
@@ -139,3 +163,38 @@ def get_screen():
     buffered_screen = BytesIO()
     screen.save(buffered_screen, format='JPEG')
     return base64.b64encode(buffered_screen.getvalue()).decode('utf-8')
+
+
+def url_history(url):
+    """Stores 10 last casted url
+
+    :param url: URL to save
+    :type url: str
+
+    :return: List of urls
+    :rtype: list
+    """
+    if url is '' or None or []:
+        pass
+    else:
```

Comment:
It’s C++ logic. Try to implement queue logic here. I think that python even dedicated type for this.